builtin/providers/google/resource_google_project_services.go

```diff
@@ -160,7 +168,9 @@ func getApiServices(pid string, config *Config) ([]string, error) {
 		return apiServices, err
 	}
 	for _, v := range svcResp.Services {
-		apiServices = append(apiServices, v.ServiceName)
+		if _, ok := ignore[v.ServiceName]; !ok {
+			apiServices = append(apiServices, v.ServiceName)
+		}
```

Comment:
Yup, and just in case anyone else is wondering, here's what the test output looks like with a DiffSuppressFunc:
```
--- FAIL: TestAccGoogleProjectServices_ignoreUnenablableServices (260.05s)
	testing.go:280: Step 0 error: After applying this step, the plan was not empty:

		DIFF:

		UPDATE: google_project_services.acceptance
		  services.#:          "10" => "9"
		  services.133405307:  "storage-component.googleapis.com" => "storage-component.googleapis.com"
		  services.2692275209: "dataproc.googleapis.com" => "dataproc.googleapis.com"
		  services.2966512281: "deploymentmanager.googleapis.com" => "deploymentmanager.googleapis.com"
		  services.3010261123: "replicapool.googleapis.com" => "replicapool.googleapis.com"
		  services.3075019877: "replicapoolupdater.googleapis.com" => "replicapoolupdater.googleapis.com"
		  services.3077910291: "resourceviews.googleapis.com" => "resourceviews.googleapis.com"
		  services.3731214611: "compute-component.googleapis.com" => "compute-component.googleapis.com"
		  services.3740470850: "container.googleapis.com" => "container.googleapis.com"
		  services.3875785048: "storage-api.googleapis.com" => "storage-api.googleapis.com"
```